src/env-inl.h

```diff
@@ -775,6 +775,10 @@ inline void AllocatedBuffer::Resize(size_t len) {
   buffer_ = uv_buf_init(new_data, len);
 }
 
+inline bool AllocatedBuffer::empty() {
```

Comment:
I think this is confusing, because for most memory chunks, one would define “empty” as “of length 0”? But see also the comment below, on a first glance this seems like it’s not a meaningful thing to add anyway…